src/ext/java/org/opentripplanner/ext/geocoder/LuceneIndex.java

```diff
@@ -65,9 +70,12 @@ public class LuceneIndex implements Serializable {
   private final SuggestIndexSearcher searcher;
   private final StopClusterMapper stopClusterMapper;
 
-  public LuceneIndex(TransitService transitService) {
+  public LuceneIndex(
+    TransitService transitService,
```

Comment:
The transit service is request scoped, is the LuceneIndex request-scoped?

Reply:
No, it isn't and I've reworked this in the newest commits.  However, I do need a few methods from TransitService. Is it ok to instantiate it myself or should I create my own wrapper around the TransitModel?